leapp/reporting/__init__.py

```diff
@@ -0,0 +1,191 @@
+from leapp.models import Model, fields
+from leapp.topics import ReportTopic
+from leapp.libraries.stdlib.api import produce
+import json
+
+
+class ReportVersions(Model):
+    topic = ReportTopic
+
+    framework = fields.String()
+    actors = fields.String()
+
+
+class Renderers(Model):
+    topic = ReportTopic
+
+    html = fields.String()
+    plaintext = fields.String()
+
+
+class Report(Model):
+
+    __non_inheritable__ = True
+
+    topic = ReportTopic
+
+    """
+    Version of framework and actors as :class:`ReportVersions`
+    """
+    versions = fields.Model(ReportVersions)
```

Comment:
No sense for having the version of the framework and actors included in each report. You rather should have one ReportVersions per execution. It's just a field of noise IMHO

Reply:
We got rid of versioning completely. Initially we were thinking about generating it in specfile but we would not have it upstream then...